### PR TITLE
Fix alignment issue in recipe creating window | Add alt text of image to image alt attr in recipe

### PR DESCRIPTION
**Pull request by hgezim**
- Fixed issue with alignment in create recipe window
- Added alt text to images.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ src/vendor-dev/
 plugins/
 node_modules/
 build/
+/nbproject/private/
```
Comment (hgezim) [inline .gitignore]: Add nbproject to .gitignore so this won't be added.

File: src/styles/zlrecipe-dlog.css [hunk 2]
```diff
@@ -21,25 +21,31 @@ body#amd-zlrecipe-uploader { }
...
+#amd-zlrecipe-uploader input[type="submit"] { -moz-border-radius: 11px; -khtml-border-radius: 11px; -webkit-border-radius: 11px; border-radius: 11px; -moz-box-sizing: content-box; border-style: solid; cursor: pointer; line-height: 13px; padding: 10px 25px; text-decoration: none; margin: 16px; text-shadow: 0 1px 0 #fff; background-color: #fff; border: 1px solid #bbb; font-size: 123.1%; font-weight: bold; }
 #amd-zlrecipe-uploader input[type="submit"]:hover { border-color: #666; }
 #amd-zlrecipe-uploader .input-error { border: 2px solid red !important; }
 #amd-zlrecipe-uploader #z_recipe_ingredients textarea.input-error { width: 585px !important; }
 #amd-zlrecipe-uploader p.error-message { color: red; font-weight: bold; }
-#amd-zlrecipe-uploader #recipe-title p.error-message { margin-left: 120px; }
\ No newline at end of file
+#amd-zlrecipe-uploader #recipe-title p.error-message { margin-left: 120px; }
+
+/*-----[sahil fixes]-----*/
```
Comment (hgezim) [inline src/styles/zlrecipe-dlog.css]: These comments aren't necessary.

File: src/views/create-update-recipe.twig [hunk 3]
```diff
@@ -80,6 +80,7 @@
             // Show it
             // Set $image src to url
             // scale image
+            //console.log(imageData.alt);
```
Comment (hgezim) [inline src/views/create-update-recipe.twig]: Clean this up.

File: src/styles/zlrecipe-dlog.css [hunk 4]
```diff
@@ -21,25 +21,31 @@ body#amd-zlrecipe-uploader { }
...
+#amd-zlrecipe-uploader #recipe-title input[type="text"] {
+    width:90%;
+}
+/*-----[/sahil fixes]-----*/
```
Comment (hgezim) [inline src/styles/zlrecipe-dlog.css]: Remove comment plz

File: src/styles/zlrecipe-print.css [hunk 5]
```diff
@@ -86,61 +86,68 @@ Copyright 2011, 2012 ZipList, Inc.
...
 
 
 /* END PRINT FORMATTING */
+/*------[sahil fixes]----------*/
+#zlrecipe-print-container img.photo{
+    max-width:100% !important;
+    height:auto;
+    display:block;
+}
+/*------[/sahil fixes]----------*/
```
Comment (hgezim) [inline src/styles/zlrecipe-print.css]: Remote comment

File: src/styles/zlrecipe-print.css [hunk 6]
```diff
@@ -86,61 +86,68 @@ Copyright 2011, 2012 ZipList, Inc.
...
-  background-position-y: -150px; }
+    background-position-y: -150px; }
 
 #zlrecipe-print-container .zrdn_zero {
-  background-position-y: -168px; }
+    background-position-y: -168px; }
 
 
 /* END PRINT FORMATTING */
+/*------[sahil fixes]----------*/
```
Comment (hgezim) [inline src/styles/zlrecipe-print.css]: Remove comment

File: src/styles/zlrecipe-print.css [hunk 7]
```diff
@@ -86,61 +86,68 @@ Copyright 2011, 2012 ZipList, Inc.
...
+#zlrecipe-print-container img.photo{
+    max-width:100% !important;
```
Comment (hgezim) [inline src/styles/zlrecipe-print.css]: Shouldn't resort to using `!important` for no good reason.

**Comment by hgezim** [comment]
Fix placing here:
add_new_post_ _dev_blog_ _wordpress

**Comment by hgezim** [comment]
Author field is kind of messed up:

add_new_post_ _dev_blog_ _wordpress

**Comment by hgezim** [comment]
You can add an image in ingredients and instructions like this:

```
ingredient 1
ingredient 2
%https://static.thespicehouse.com/images/image/30/large_red-hot-chili.jpg
```

Print preview resizing isn't happening in this case:
about_blank

**Review by hgezim — COMMENTED** [review]
A couple fixes. Also looks like there's a conflict.

File: .gitignore (hunk 1 above)
Comment (sahilbabu) [reply .gitignore]: Okay i got it

File: src/styles/zlrecipe-dlog.css (hunk 4 above)
Comment (sahilbabu) [reply src/styles/zlrecipe-dlog.css]: Okay i will

File: src/views/create-update-recipe.twig (hunk 3 above)
Comment (sahilbabu) [reply src/views/create-update-recipe.twig]: Okay i will

File: src/styles/zlrecipe-print.css (hunk 5 above)
Comment (sahilbabu) [reply src/styles/zlrecipe-print.css]: I got it

**Pull request merged**